README.md

```diff
@@ -1,14 +1,9 @@
-# Intel(R) oneAPI Collective Communications Library (oneCCL)
-
+# oneccl
+Intel(R) oneAPI Collective Communications Library
```

Comment:
Intel(R) oneAPI Collective Communications Library (oneCCL) - this is approved product name. Please start using full product name and later in the text you can use oneCCL only.